modules/node/handler/cms_node_pagination_builder.e

```diff
@@ -0,0 +1,44 @@
+note
+	description: "Paginator builder for CMS nodes."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	CMS_NODE_PAGINATION_BUILDER
+
+inherit
+
+	CMS_PAGINATION_BUILDER [CMS_NODE]
+
+	CMS_NODE_HANDLER
```

Comment:
I disagree with that inheritance.
for me a pagination builder is not a CMS_HANDLER.

if you need node_api, just pass it via the creation procedure.